neuromation/cli/job.py

```diff
@@ -80,10 +80,18 @@ def _get_neuro_mountpoint(username: str) -> str:
     return f"{ROOT_MOUNTPOINT}/{username}"
 
 
+def _read_lines(env_file: str) -> Iterator[str]:
+    with open(env_file, "r") as ef:
+        lines = ef.read().splitlines()
```

Comment:
Minor: `ef.readline()` can be used here.
I don't care too much though because the file with env vars is expected to be short, the memory footprint is not a problem here.